clang/include/clang/Driver/Options.td

```diff
@@ -6911,6 +6911,10 @@ def fsycl_fp32_prec_sqrt : Flag<["-"], "fsycl-fp32-prec-sqrt">, Group<sycl_Group
   Visibility<[ClangOption, CC1Option]>,
   HelpText<"SYCL only. Specify that single precision floating-point sqrt is correctly rounded.">,
   MarshallingInfoFlag<CodeGenOpts<"SYCLFp32PrecSqrt">>;
+def fsycl_use_builtins_for_integration : Flag<["-"], "fsycl-use-builtins-for-integration">,
```

Comment:
@premanandrao, is there an expectation for users to use this option?  If so, this should be reviewed by the Options Team WG.  If not, `HelpHidden` would be appropriate here to reduce exposure.

Reply:
Thanks for that!   At this time, we do not want uses to use this option.  This is mainly to help reintegrate upstreamed changes back here.

I will make the change.